modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/common/I18n/Language.json

```diff
@@ -385,8 +385,9 @@
 	"removing": "Removing",
 	"renew": "Renew",
 	"renew-key": "Renew Key",
-	"renew-x-activation-keys": "Renew {0} Activations Key(s)",
-	"renew-x-key": "Renew {0} Key(s)",
+	"renew-x-activation-key": "Renew Activations Key",
```

Comment:
There is still an `x` in the lang key, but you arent using it. According to Brian's comment, there should probably be 2 of these as well, one for singular and one for plural versions.

Also plural should be `Renew Activation Keys` rather than `Renew Activations Key`

Reply:
adjusted! thanks @kylembischof

Reply:
can you also please add the ticket number to your commit message. :+1: